app/facades/quote_facade.rb

```diff
@@ -0,0 +1,15 @@
+class QuoteFacade
+  def self.top_10_query_quotes(search_query)
+    quotes = QuoteService.get_search_quotes_api_call(search_query)
+
+    quotes[:results].map do |quote|
+      Quote.new(quote)
+    end.(0..9) #might need to limit another way to include param if less than 10
```

Comment:
Instead of creating poros for EVERY quote and then only taking the first 10, I would instead do something like: `quotes[:results].first(10).map` so you are only making objects out of the ones you plan to display. If you had thousands of quotes, this current setup could get very slow.